spdx-utils/src/test/kotlin/SpdxUtilsTest.kt

```diff
@@ -19,16 +19,36 @@
 
 package com.here.ort.spdx
 
+import com.here.ort.utils.safeDeleteRecursively
+
+import io.kotlintest.TestCase
+import io.kotlintest.TestResult
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldNot
+import io.kotlintest.shouldNotBe
 import io.kotlintest.matchers.endWith
 import io.kotlintest.matchers.startWith
 import io.kotlintest.matchers.string.beBlank
-import io.kotlintest.should
-import io.kotlintest.shouldBe
-import io.kotlintest.shouldNot
 import io.kotlintest.specs.WordSpec
 
+import java.io.File
+
 class SpdxUtilsTest : WordSpec() {
 
+    private var tempDir: File? = null
+
+    private fun setupTempFile(filename: String, content: String) {
+        if(tempDir == null) {
+            tempDir = createTempDir(SpdxUtilsTest::class.simpleName!!)
+        }
+        File(tempDir, filename).writeText(content)
```

Comment:
Nit: Missing blank line before this one.

Reply:
In general, the slightly simpler pattern we use in existing tests is:

    private lateinit var outputDir: File

    override fun beforeTest(testCase: TestCase) {
        outputDir = createTempDir()
    }

    override fun afterTest(testCase: TestCase, result: TestResult) {
        outputDir.safeDeleteRecursively(force = true)
    }

Can we use that here, too?

Reply:
My pattern not only creates a dir but also a file so it cannot be used exactly like above. Moreover I prefer using the if for execution speed of tests not needing the temp dir.

Reply:
That's a hard decision imo, I tend to prefer the speed option here - but I would change it as well to your above example. what do you think?